sidechain/consensus/aura/src/test/block_importer_tests.rs

```diff
@@ -116,12 +120,21 @@ fn signed_block(
 ) -> SignedSidechainBlock {
 	let state_update = empty_encrypted_state_update(state_handler);
 
-	SidechainBlockBuilder::default()
-		.with_timestamp(duration_now().as_millis() as u64)
-		.with_parentchain_block_hash(parentchain_header.hash())
+	let header = SidechainHeaderBuilder::default()
 		.with_parent_hash(H256::default())
 		.with_shard(shard())
+		.build();
+
+	let block_data = SidechainBlockDataBuilder::default()
+		.with_timestamp(duration_now().as_millis() as u64)
```

Comment:
If you have some time left: We have a crate `itp-time-utils` that has a utility function for this: `now_as_u64()`